src/docs/product/integrations/spikesh/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Spike.sh
+sidebar_order: 1
+redirect_from:
+  - /workflow/integrations/spikesh/
+  - /workflow/gintegrations/lobal-integrations/spikesh/
```

Comment:
We only need to add redirects if we are moving content. I believe this is new, so no redirects are required.